### ISSUE TITLE
Add Dash Support

### ISSUE THREAD

**Issue by pento** [PonyEdit/PonyEdit#1]
When Dash is installed, pressing Fn+F1 should search Dash for whatever is selected.

Dash provides extension -> keyword mapping, to make searches more accurate.

http://kapeli.com/dash_plugins

**Comment by pento** [comment]
Thing that doesn't work:

``` cpp
QDesktopServices::openUrl(QUrl("dash-plugin://keys=php&amp;query=print"));
```

**Comment by pento** [comment]
Fixed in a738c683b84c35b95c0d926528b84cb7faa0b109.
